app/views/groups/_form.html.erb

```diff
@@ -1,9 +1,15 @@
 <%= form_for(@group) do |f| %>
-<% if @group.errors.any? %>
-  <div class="error_explanation">
-    <%= t('.error_explanation') %>
-  </div>
-<% end %>
+  <% if @group.errors.any? %>
+    <div class="error_explanation">
+      <h2><%= pluralize(@group.errors.count, 'error') %>
+        prohibited this group from being saved:</h2>
```

Comment:
IMO, I’d probably humanize this by saying something in 'we' language: We can’t save this group because:

*
*

We could also just go through all the errors later and revise the language.